plantseg/resources/models_zoo.yaml

```diff
@@ -55,7 +55,7 @@ confocal_2D_unet_ovules_ds2x:
   description: "2D Unet trained on z-slices of confocal images of Arabidopsis Ovules (1/2-resolution in XY) with BCEDiceLoss."
   dimensionality: "2D"
   modality: "confocal"
-  recommended_patch_size: [ 80, 170, 170 ]
+  recommended_patch_size: [ 1, 170, 170 ]
```

Comment:
here we should use `[1, 256, 256]`

Reply:
Done, plus [ 80, 170, 170 ] > [ 80, 160, 160 ], because 170 sounds like a weird choice to me :)